recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -4,11 +4,11 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 # Renesas R-Car
 ################################################################################
 
-XEN_REL_rcar = "4.10"
+XEN_REL_rcar = "4.12"
 PV = "${XEN_REL}.0+git${SRCPV}"
 SRCREV_rcar = "${AUTOREV}"
 
-SRC_URI_rcar = "git://github.com/xen-troops/xen.git;protocol=https;branch=master"
+SRC_URI_rcar = "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.12rc-migration"
```

Comment:
I would rather think of moving current master, which is 4.10, to some tag/branch making sure all other dependent products use that. After that moving master to 4.12

Reply:
xen-4.12rc-migration is a current feature branch. Actually it's a mock for now.